apiserver/facades/client/imagemetadatamanager/metadata_test.go

```diff
@@ -111,26 +108,34 @@ func (s *metadataSuite) TestFindOrder(c *gc.C) {
 		{ImageId: customImageId2, Priority: 20},
 		{ImageId: publicImageId, Priority: 15},
 	})
-	s.assertCalls(c, controllerTag, findMetadata)
+	s.assertCalls(c, findMetadata)
 }
 
 func (s *metadataSuite) TestSaveEmpty(c *gc.C) {
-	ctrl, _ := s.setupAPI(c)
-	defer ctrl.Finish()
+	defer s.setupAPI(c).Finish()
+
+	s.modelInfoService.EXPECT().GetModelInfo(gomock.Any()).Return(
+		coremodel.ReadOnlyModel{
+			CloudRegion: "",
```

Comment:
should we have an actual region value here? And not just empty?

Reply:
We should